fhir-install/Dockerfile

```diff
@@ -18,7 +18,6 @@ RUN unzip -qq /tmp/fhir-server-distribution.zip -d /tmp && \
     mv /tmp/fhir-server-dist/tools /opt/fhir-server/tools
 COPY src/main/docker/ibm-fhir-server/bootstrap.properties /opt/ol/wlp/usr/servers/defaultServer/
```

Comment:
still ibm-fhir-server?

Reply:
I added a commit to this PR for this one.
I left the `ibm-fhir-*` names in just two places:
1. the migration tests (so that we can continue using a mix of old and new); and
2. the security workflow (because I havn't touched this one much and it probably needs some dedicated attention)

We've decided to use GitHub Packages (ghcr) for our docker images moving forward,
so we'll have more release automation work to make that happen.  I'll open a separate issue for that part of it.

Reply:
this is now https://github.com/LinuxForHealth/FHIR/issues/3845